src/Module/Api/Twitter/Lists/Ownership.php

```diff
@@ -48,8 +48,11 @@ protected function rawContent(array $request = [])
 
 		// loop through all circles
 		$lists = [];
-		foreach ($circles as $circle) {
-			$lists[] = $this->friendicaCircle->createFromId($circle['id']);
+
+		if (is_iterable($circles)) {
```

Comment:
Please use `Database->selectToArray` on line 47 instead of adding a condition.

Reply:
Done.